tbd.schema.json

```diff
@@ -441,6 +469,106 @@
           "title": "Global state",
           "type": "string"
         },
+        "ua": {
+          "title": "UA' assessment",
+          "type": "object",
+          "properties": {
+            "bloc1": {
+              "title": "heating setpoints >= 18°C",
+              "type": "object",
+              "properties": {
+                "summary": {
+                  "type": "string"
+                },
+                "walls": {
+                  "type": "string"
+                },
+                "roofs": {
+                  "type": "string"
+                },
+                "floors": {
+                  "type": "string"
+                },
+                "doors": {
+                  "type": "string"
+                },
+                "windows": {
+                  "type": "string"
+                },
+                "skylights": {
+                  "type": "string"
+                },
+                "rimjoists": {
+                  "type": "string"
+                },
+                "parapets": {
+                  "type": "string"
+                },
+                "trim": {
+                  "type": "string"
+                },
+                "corners": {
+                  "type": "string"
+                },
+                "grade": {
+                  "type": "string"
+                },
+                "other": {
+                  "type": "string"
+                }
+              },
+              "additionalProperties": false,
+              "required": ["summary"]
+            },
+            "bloc2": {
+              "title": "heating setpoints >= 18°C",
+              "type": "object",
+              "properties": {
+                "summary": {
+                  "type": "string"
+                },
+                "walls": {
+                  "type": "string"
+                },
+                "roofs": {
+                  "type": "string"
+                },
+                "floors": {
+                  "type": "string"
+                },
+                "doors": {
+                  "type": "string"
+                },
+                "windows": {
+                  "type": "string"
+                },
+                "skylights": {
+                  "type": "string"
+                },
+                "rimjoists": {
+                  "type": "string"
+                },
+                "parapets": {
+                  "type": "string"
+                },
+                "trim": {
+                  "type": "string"
+                },
+                "corners": {
+                  "type": "string"
+                },
+                "grade": {
+                  "type": "string"
+                },
+                "other": {
+                  "type": "string"
+                }
+              },
+              "additionalProperties": false,
+              "required": ["summary"]
+            }
```

Comment:
Want to add a `bloc3` just for completeness?

Reply:
For the moment, I only have 2x categories to consider: "above 18°C" vs "below 18°C". If ever (in the future) a need for a 3rd (or 4th or nth) option pops up, I think I would revise the strategy and switch over to an (arbitrarily-sized) array of blocks (or bins).